experiments/ClimaEarth/setup_run.jl

```diff
@@ -82,87 +83,18 @@ include("user_io/postprocessing.jl")
 include("user_io/coupler_diagnostics.jl")
 
 """
-    solve_coupler!(cs)
+    CoupledSimulation(config_file)
+    CoupledSimulation(config_dict)
 
-This function performs the coupling loop, which is the main part of the simulation.
-It runs the component models sequentially for one coupling timestep (`Δt_cpl`) at a time,
-and exchanges combined fields and calculates fluxes using the selected turbulent fluxes option.
-Note that we want to implement this in a dispatchable function to allow for
-other forms of timestepping (e.g. leapfrog).
+Set up a `CoupledSimulation` as prescribed by the given input.
```

Comment:
This could be a bit more descriptive, e.g.:
```suggestion
Set up a `CoupledSimulation` as prescribed by the given input.
This struct is defined in the Interfacer module and contains all
information about component models, diagnostics, timestepping,
output directories, etc needed to run a coupled simulation.
```